code-for-canada/project-thundercat
# [Refactor] Tab styles pulled out of App.css

# Description

Pulled Tab and TabNavigation styles out of App.css. Removed some unused css

## Type of change

Please delete options that are not relevant.

- Code cleanliness or refactor

## Screenshot

Looks the same as PR #94 

# Testing

Manual steps to reproduce this functionality:

1.  Go to Start Emib
2. Navigate through to TestTabs
3. Verify that pages look the same as they did previously

Screenshot of unit tests run passing:

# Checklist

Applicable for all code changes.
